stylesheets/components/modal.css

```diff
@@ -0,0 +1,141 @@
+/* Alert Modal */
+
+.modal-alert-container,
+.modal-simple-container {
+  background-color: var(--white);
+  padding: 2rem;
+}
+
+.modal-alert-container {
+  width: min(100%, 35rem);
+}
+
+.modal-alert-header,
+.modal-simple-header,
+.modal-confirmation-header {
+  color: var(--secondary-color);
+  font-size: 2rem;
+  font-weight: 600;
+}
+
+.modal-alert-body {
+  font-size: 1.6rem;
+  font-weight: 500;
+  padding: 2rem 0rem;
+  color: var(--black-500);
+}
+
+.modal-alert-action {
+  display: flex;
+  justify-content: flex-end;
+  flex-wrap: wrap;

```

Comment:
you can also use flex flow  :row wrap to reduce code it is horthand property

Reply:
Okay added

Reply:
okay